Library with TS doesn't work

<!--
Thank you very much for helping improving this library
-->

### Description
When react-pixi is used in TS project, compiling is not working. I'm receiving error:
`ERROR in ../node_modules/@inlet/react-pixi/index.d.ts:3:28 - error TS7016: Could not find a declaration file for module 'react-reconciler'. 'D:/test/game/frontend/node_modules/react-reconciler/index.js' implicitly has an 'any' type.
  Try `npm install @types/react-reconciler` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-reconciler';`

3 import { Reconciler } from 'react-reconciler';
                             ~~~~~~~~~~~~~~~~~~`


### Steps to reproduce

1. https://repl.it/repls/DecisiveBadComputation
2.

### Additional info

- `@inlet/react-pixi` version: 5.1.5
- `React` version: 16.8.2
- `ReactDOM` version: 16.8.2
- `PIXI` version: 1.1.6

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,6 +1,42 @@
 import * as PIXI from 'pixi.js';
 import * as React from 'react';
-import { Reconciler } from 'react-reconciler';
+
+// Reconciler API
+interface Reconciler<Instance, TextInstance, Container, PublicInstance> {
+  updateContainerAtExpirationTime(
+      element: any,
+      container: any,
+      parentComponent: React.Component<any, any> | null | undefined,
+      expirationTime: any,
+      callback: () => void | null | undefined,
+  ): any;
+  createContainer(
+      containerInfo: any,
+      isConcurrent: boolean,
+      hydrate: boolean,
+  ): any;
+  updateContainer(
+      element: any,
+      container: any,
+      parentComponent: React.Component<any, any> | null | undefined,
+      callback: () => void | null | undefined,
+  ): any;
+  flushRoot(root: any, expirationTime: any): void;
+  requestWork(root: any, expirationTime: any): void;
+  computeUniqueAsyncExpiration(): any;
+  batchedUpdates<A>(fn: () => A): A;
+  unbatchedUpdates<A>(fn: () => A): A;
+  deferredUpdates<A>(fn: () => A): A;
+  syncUpdates<A>(fn: () => A): A;
+  interactiveUpdates<A>(fn: () => A): A;
+  flushInteractiveUpdates(): void;
+  flushControlled(fn: () => any): void;
+  flushSync<A>(fn: () => A): A;
+  getPublicRootInstance(container: any): React.Component<any, any> | PublicInstance | null;
+  findHostInstance(component: object): PublicInstance | null;
+  findHostInstanceWithNoPortals(component: any): PublicInstance | null;
+  injectIntoDevTools(devToolsConfig: any): boolean;
+}
 
 // private
 declare namespace _ReactPixi {

